### PR TITLE
Upgrade styleguidist v9

### LINKED ISSUE #420: Upgrade styleguidist to latest

### PR DESCRIPTION
**Pull request by cshepherd**
Resolves #420 
Impact: **minor**  
Type: **docs**

## Component
Upgraded react-styleguidist to 9.1.11
Upgraded webpack to 4.35.3 (4.x is dependency for latest styleguidist)

Moved font, Stripe script, and meta tags from deprecated HTML template into styleguidist.config.js template JSON area.

As of v9 in Styleguidist, all components (other than the one currently under discussion) are no longer available in the global (window) context in the .md files, hence new requires to pull the components in manually.

### CODE REVIEW

**Comment by rc-publisher** [comment]
🚀 Preview deployed

Built with commit d84c7aff2557075d2343c129f5c8ee5c4169d834

https://deploy-preview-421--stoic-hodgkin-c0179e.netlify.com

**Comment by machikoyasuda** [comment]
Almost forgot how 🆒 😎 🆒  it is to have Netlify branch deploys!

**Comment by rc-publisher** [comment]
🚀 Preview deployed

Built with commit d994243985a4e12a91c3a6973160fa9897424f36

https://deploy-preview-421--stoic-hodgkin-c0179e.netlify.com

File: package/src/components/ErrorsBlock/v1/ErrorsBlock.md [hunk 1]
```diff
@@ -3,7 +3,9 @@
 The `ErrorsBlock` is used to provide helpful information to the user when a form is invalid. The block takes an array of errors that are displayed with an optional icon.
 
 ```jsx
-const TwoColumnExamples = require("../../../../../styleguide/src/components/TwoColumnExamples").default;
+import Field from "../../Field/v1/Field"
+import TextInput from "../../TextInput/v1/TextInput"
+;const TwoColumnExamples = require("../../../../../styleguide/src/components/TwoColumnExamples").default;
```
Comment (machikoyasuda) [inline package/src/components/ErrorsBlock/v1/ErrorsBlock.md]: Error here caused by `;const`: https://deploy-preview-421--stoic-hodgkin-c0179e.netlify.com/#/Base%20Components?id=errorsblock

The `;` semicolons got pushed down into the 2nd row for a lot of these. Putting them back on the same line as the `import` should fix these!

**Comment by cshepherd** [comment]
After some dependency-wrangling, I did finally come up against one (apparently) intractable snyk issue arising from libnpx@10.2.0. Since 'snyk wizard' indicated there was no available mitigation, I added a .snyk file to the repo to ignore the issue for now.

**Pull request merged**